### PR TITLE
Chronograf 1.8.10

### PR DESCRIPTION
**Pull request by kelseiv**
Closes the following issues:
https://github.com/influxdata/docs-v2/issues/2141
https://github.com/influxdata/docs-v2/issues/2084

### CODE REVIEW

**Review by russorat — APPROVED** [review]
LGTM

**Review by timhallinflux — CHANGES_REQUESTED** [review]
Is there a section of the docs that describes the meta query templates?  because we've added DROP SERIES, DELETE and DROP MEASUREMENT.  I can't seem to find a description of these in the Data Explorer walk thru.

**Comment by kelseiv** [comment]
> Is there a section of the docs that describes the meta query templates? because we've added DROP SERIES, DELETE and DROP MEASUREMENT. I can't seem to find a description of these in the Data Explorer walk thru.

@timhallinflux, we mention "Metaquery Templates" in step 4 of [Explore data with InfluxQL](https://docs.influxdata.com/chronograf/v1.8/guides/querying-data/#explore-data-with-influxql)...however, it appears something was inadvertently deleted. Created [an issue](https://github.com/influxdata/docs-v2/issues/2164) to resolve this and related docs issues.

File: content/chronograf/v1.8/about_the_project/release-notes-changelog.md [hunk 1]
```diff
@@ -8,6 +8,33 @@ menu:
...
+
+- Add the ability to set the active InfluxDB database and retention policy for InfluxQL commands. Now, in Chronograf Data Explorer, if you select a metaquery template (InfluxQL command) that requires you to specify an active database, such as `DROP MEASUREMENT`, `DROP SERIES FROM`, and `DELETE FROM`, the `USE` command is prepended to your InfluxQL command as follows:
+
+```
+USE "db_name"; DROP MEASUREMENT "measurement_name"
+USE "db_name"; DROP SERIES FROM "measurement_name" WHERE "tag" = 'value'
+USE "db_name"; DELETE FROM "measurement_name" WHERE "tag" = 'value' AND time < '2020-01-01'
+```
+
+- Upgrade to Axios 0.21.1.
```
Comment (timhallinflux) [inline content/chronograf/v1.8/about_the_project/release-notes-changelog.md:23]: move to Bug Fix/maintenance?

**Pull request merged**